bootstrap/schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://spacemesh.io/checkpoint.schema.json.1.0",
+  "title": "epoch data",
+  "description": "epoch data for bootstrapping and fallback",
+  "type": "object",
+  "required": ["version", "data"],
+  "properties": {
+    "version": {
+      "description": "version of the checkpoint file. should be compatible schema's $id",
+      "type": "string"
+    },
+    "data": {
+      "description": "the signed portion of this file",
+      "type": "object",
+      "required": ["id", "epochs"],
+      "properties": {
+        "id": {
+          "type": "integer",
+          "exclusiveMinimum": 0
+        },
+        "epochs": {
```

Comment:
Is this an array of all epochs since genesis?

Reply:
most likely we need one epoch for bootstrap fallback purposes. keeping it an array for now to leave options open.